index.bs

```diff
@@ -3093,6 +3093,43 @@ This [=registration extension=] and [=authentication extension=] enables use of
                     01           -- Subitem 3: CBOR short for Matcher Protection Type Software
     </pre>
 
+## Authenticator Biometric Criteria Extension (biometricBound) ## {#sctn-authenticator-biometric-criteria-extension}
+
+This [=registration extension=] allows a [=[RP]=] to specify the desired performance of a biometric authenticator
+that could be leveraged when creating the credential. 
+
+: Extension identifier
+:: `biometricBound`
+
+: Client extension input
+:: Biometric performance bounds:
+
+    <pre class="idl">
+    dictionary biometricCriteria{
+        float FAR;
+        float FRR;
+        };
+    </pre>
+
+    The FAR is the maximum false acceptance rate for a biometric authenticator allowed by the [=[RP]=].
+
+    The FAR is the maximum false rejection rate for a biometric authenticator allowed by the [=[RP]=].
+
+: Client extension processing
+:: This extension can only be used during {{CredentialsContainer/create()}}. If the client supports the Authenticator Biometric Criteria
+    Extension and biometric authenticators are available, it MUST use the first available biometric authenticator whose FAR and FRR match the bounds as provided.
```

Comment:
we perhaps ought to define "biometric authenticators" up in terminology section, and then use `[=biometric authenticators=]` here.